tslint-config-5minds/tslint.json

```
@@ -54,74 +75,154 @@
     "unified-signatures": true,
 
     "await-promise": false,
+    "ban-comma-operator": true,
+    "ban": {
+      "options": [
+        {"name": ["global", "isFinite"], "message": "Please use Number.isFinite instead"},
+        {"name": ["self", "isFinite"], "message": "Please use Number.isFinite instead"},
+        {"name": ["window", "isFinite"], "message": "Please use Number.isFinite instead"},
+        {"name": ["global", "isNaN"], "message": "Please use Number.isNaN instead"},
+        {"name": ["self", "isNaN"], "message": "Please use Number.isNaN instead"},
+        {"name": ["window", "isNaN"], "message": "Please use Number.isNaN instead"},
+        {"name": ["__defineGetter__"], "message": "Please use Object.defineProperty instead."},
+        {"name": ["__defineSetter__"], "message": "Please use Object.defineProperty instead."},
+        {"name": ["Math", "pow"], "message": "Use the exponentiation operator (**) instead."}
+      ]
+    },
     "curly": true,
     "forin": false,
-    "import-blacklist": [true, "rxjs", "lodash"],
+    "function-constructor": true,
+    "import-blacklist": {
+      "options": ["rxjs", "lodash"]
```

Falls man hier Wünsche äußern kann, würde ich hier auch noch liebend gerne `ramda` drauf setzen. 🙂 

Dieses "Tool" ist eine noch größere Katastrophe als lodash, wie mich meine Zeit bei Huf gelehrt hat.
Ich denke @5ebastianMeier wird mir da zustimmen. 

In `@essential-projects/tslint-config` steht es schon länger auf der Blacklist.

---

Da der PR zum updaten der Regeln gilt, und wir mittlerweile mehr Erfahrung gesammelt haben, als wir vor 2 Jahren hatten: Ja, es können wünsche geäußert werden, wenn sie (wie hier) entspreched verargumentiert sind :)

---

Dann wünsch ich mir das jetzt 😬 

```suggestion
      "options": ["ramda", "rxjs", "lodash"]
```

---

Glaub ramda ist nicht schuld. Sachen wie Currying, Pipes und Funkionskompositionen vermisse ich regelmäßig bei js. Kann zu stabileren und lesbareren Code führen.

Wär aber sehr ungewohnt in den bisherigen Codebasen und machts schwierig für Leute ohne funktionalen Bezug, deshalb vllt nicht verkehrt auf der Liste.